homeassistant/components/sensor/rfxtrx.py

```diff
@@ -24,6 +24,9 @@
 
 from homeassistant.const import (TEMP_CELCIUS)
 from homeassistant.helpers.entity import Entity
+import homeassistant.components.rfxtrx as rfxtrx
+from RFXtrx import SensorEvent
+from homeassistant.util import slugify
 
 REQUIREMENTS = ['https://github.com/Danielhiversen/pyRFXtrx/archive/' +
                 'ec7a1aaddf8270db6e5da1c13d58c1547effd7cf.zip#RFXtrx==0.15']
```

Comment:
I think this should work: https://github.com/Danielhiversen/pyRFXtrx/archive/0.2.zip

(badele: you have write access to the pyRFXtrx lib )